airbyte-integrations/connectors/source-harvest/source_harvest/spec.yaml

```diff
@@ -0,0 +1,54 @@
+documentationUrl: https://help.getharvest.com/api-v2/
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  type: object
+  required:
+    - account_id
+    - start_date
+    - auth_type
```

Comment:
spec file does not align with the previous version.
credentials should be a property of type `object` with a required list depending on its type.

Reply:
Using the connector builder changed the format of the spec. To keep it compatible with the connector builder for future updates, I felt like it is preferred to force the change now.

Reply:
Reverted config to old format and added a `config_migration.py`.